src/query/graphite/graphite/identify_test.go

```diff
@@ -0,0 +1,44 @@
+package graphite
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	statsdStat    = "dispatch.production.san_francisco.uberx.drivers"
+	malformedStat = "dispatchproductionsan_franciscouberxdrivers"
+	dashedString  = "some-other-delimiter"
+)
+
+func TestExtractNthEmptyMetric(t *testing.T) {
+	assert.Equal(t, "", ExtractNthMetricPart("", 0))
+	assert.Equal(t, 0, CountMetricParts(""))
+}
+
+func TestExtractNthMetricPartNoDots(t *testing.T) {
+	nodots := "dispatchproductionsan_franciscouberxdrivers"
```

Comment:
Similar, need to change both these two stats to more test data-ish looking names.